libcodechecker/server/client_db_access_handler.py

```diff
@@ -20,6 +27,10 @@
 from codeCheckerDBAccess import constants
 from codeCheckerDBAccess.ttypes import *
 
+from libcodechecker import generic_package_context
+from libcodechecker import suppress_handler
+from libcodechecker.analyze import plist_parser
```

Comment:
Back when the subcommands and the whole of `libcodechecker` was split and reorganised, imports which violated subcommand boundaries (such as `server` depending on a class defined in `analyze`) were marked with the same(-esque) _TODO_.

It's a marker for us that there are architectural mishaps in CodeChecker. It's not a problem as long as CodeChecker is a single and whole being in itself, but the moment we want to create separate builds from CodeChecker, e.g. one that can **only** log and analyze, or one that is **only** a webserver, these imports make easy creation of such packages problematic, and these markers show us that _"Yeah, there should one day be a refactoring here!"_. (There were discussion about this package separation around 7-9 months ago, but that discussion died due to the plethora of other features to be made.)